### ISSUE TITLE
Accessibility check

### ISSUE THREAD

**Issue by Faithchidindu** [oppia/lessons-team#684]
![oppia 2](https://user-images.githubusercontent.com/127211589/229183700-cca398f8-542a-405e-9e69-03dc75a11f34.jpg)
![oppia 1](https://user-images.githubusercontent.com/127211589/229185493-06cc16f5-ccc4-4c44-9374-5f27f9b501fe.jpg)

The “stories you can play” text was not quite visible. The learner might struggle to read it. The text “stories you can play’ was given a white colour on a grey background so it did not pass the WCAG accessibility check.
It would be better if the text was given a black colour. 
White text should be used on a dark background, dark text should be used on white background inorder to make the text easy to read

There is inconsistency in the UI theme. Some of the screens are in light mode while some are in dark mode. There is no uniformity. The user interface will look much better if the user is allowed to switch between light and dark themes. This creates a sense of uniformity, reliability and control
